src/components/Reciter/ReciterInfo.tsx

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable @next/next/no-img-element */
+import useTranslation from 'next-translate/useTranslation';
+import { useDispatch } from 'react-redux';
+
+import PlayIcon from '../../../public/icons/play-arrow.svg';
+import Button from '../dls/Button/Button';
+import { StationState, StationType } from '../Radio/types';
+
+import styles from './ReciterInfo.module.scss';
+
+import { playFrom } from 'src/redux/slices/AudioPlayer/state';
+import { setRadioStationState } from 'src/redux/slices/radio';
+import { getImageCDNPath } from 'src/utils/api';
+import { getRandomChapterId } from 'src/utils/chapter';
+import { logEvent } from 'src/utils/eventLogger';
+import Reciter from 'types/Reciter';
+
+const BIO_SAMPLE =
```

Comment:
currently bio is a bit too long, and using html. using this for a preview for now